### DISCUSSION TITLE
[firebase_database] Realtime Database seems to have been edited out of the Flutter Fire Roadmap :(

### DISCUSSION THREAD

**Discussion by animeshjain** [firebase/flutterfire#4031] [Q&A]
I for one was very much awaiting work on the Realtime Database web compatibility as well. Has it been dropped because not many people are using it any more?

**Comment by deepak786** [comment]
@animeshjain PR https://github.com/FirebaseExtended/flutterfire/pull/6952 is already in Progress for web support.
